vme/src/db_file.cpp

```diff
@@ -636,18 +636,18 @@ unit_fptr *bread_func(CByteBuffer *pBuf, ubit8 version, unit_data *owner, int st
     {
         if (fptr)
         {
-            fptr->next = new EMPLACE(unit_fptr) unit_fptr;
-            fptr = fptr->next;
+            fptr->setNext(new EMPLACE(unit_fptr) unit_fptr);
+            fptr = fptr->getNext();
         }
         else
         {
             head = new EMPLACE(unit_fptr) unit_fptr;
             fptr = head;
         }
 
-        fptr->index = pBuf->ReadU16(&g_nCorrupt);
+        fptr->readFunctionPointerIndexFrom(*pBuf, g_nCorrupt);
```

Comment:
I would really like this to work the same way as for bread() for affect. It's too confusing reading it one function at a time. Maybe that's in the next PR